packages/template-ui/channel-overrides/wikipedia/options.json

```diff
@@ -0,0 +1,9 @@
+{
+  "displayLogoInHeader": true,
+  "hasSectionsSearch": false,
+  "hasCarousel": false,
+  "hasFilters": false,
+  "displayHeroContent": true,
+  "isEndlessCurated": true,
```

Comment:
We already have a similar "isEndlessApp" option. Let's confirm with Joana if the 2 options are needed, or are the same.